internal/testrunner/reporters/xunit.go

```diff
@@ -0,0 +1,96 @@
+package reporters
+
+import (
+	"encoding/xml"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/testrunner"
+)
+
+func init() {
+	testrunner.RegisterReporter(XUnitReporter, ReportXUnit)
+}
+
+const (
+	// XUnitReporter reports test results in the xUnit format
+	XUnitReporter testrunner.TestReporter = "xUnit"
+)
+
+type testSuites struct {
+	XMLName xml.Name    `xml:"testsuites"`
+	Suites  []testSuite `xml:"testsuite"`
+}
+type testSuite struct {
```

Comment:
Some xml annotations have first uppercase letter, some don't. Is it on purpose?

Reply:
No, not on purpose. Good catch, will fix. Thanks!

Reply:
Fixed in 309a37c.